### ISSUE TITLE
Why 500 meters proximity?

### ISSUE THREAD

**Issue by oleiba** [MohGovIL/hamagen-react-native#127]
Isn't 500 meters as proximity radius a bit too much?
configured here:
https://github.com/MohGovIL/hamagen-react-native/blob/master/src/config/default_config.json#L9

I believe sick people's locations are extracted based on people's memory and not accurate GPS sampling.
However, 500 meters is still quite large, especially assuming the app's user has GPS on with good accuracy. 
May trigger lots of false positives.

**Comment by feinsteinben** [comment]
The accuracy of the GPS has nothing to do with that. The problem is with the accuracy of the data from MoH.
After the app detects a potential match, it asks the user something like "have you been to place X at time Y?", where X and Y are very specific, and the user can mark "Yes" or "No". This should eliminate most false-positives.

**Comment by oleiba** [comment]
>The accuracy of the GPS has nothing to do with that. The problem is with the accuracy of the data from MoH.

Well, that's why I said: "I believe sick people's locations are extracted based on people's memory and not accurate GPS sampling". I assume it is extracted by an epidemiological investigation.  Having the _user_'s GPS location on, definitely has something to do with that, because otherwise, you would need even a larger radius because both the sick person _and_ the user's locations are inaccurate. 

> After the app detects a potential match, it asks the user something like "have you been to place X at time Y?", where X and Y are very specific, and the user can mark "Yes" or "No". This should eliminate most false-positives.

Strange, because this article pops-up a few incidents reporting proximity of 100s of meters:
https://www.globes.co.il/news/article.aspx?did=1001323611.

**Comment by DanielMOH** [comment]
Guys, thanks for your comments!
We've launched the application with a range setting of 500 meters, because we wanted to ensure that we don't miss any potential contact events (avoiding false negatives). We were aware of the fact that it might cause users to get "false" (more accurately - "irrelevant") notifications, since it's easy for them to check the description and choose to ignore it.
During the weekend we found out that for some locations, this setting (combined with another issue of time ranges) caused many irrelevant notifications, so we lowered the range to 150 meters.
In the [upcoming release](https://github.com/MohGovIL/hamagen-react-native/releases/tag/1.1.1-qa.1) we will handle it in a more accurate way, by allowing a specific range per infected location.
